libs/commands/publish/src/index.ts

```diff
@@ -973,14 +977,25 @@ class PublishCommand extends Command {
     };
     process.on("log", logListener);
 
+    let queue;
+    if (this.options.throttle) {
+      queue = new TailHeadQueue(
+        this.options.throttleSize !== undefined ? this.options.throttleSize : 25,
```

Comment:
These default values are easy to miss, please extract them to name `const`s

Reply:
Wasn't sure where to declare them as file-global consts do not appear to be a thing in the project so I just added the const declaration right before they are used.